### PR TITLE
Upgrade dependancy

### PR DESCRIPTION
**Pull request by rudywaltz**

### CODE REVIEW

**Review by peter-vasarhelyi — APPROVED** [review]

**Review by boristomic — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -25,34 +25,35 @@
...
-    "node": "10.16.3"
+    "node": ">=10.18.0 <13.0.0",
+    "yarn": ">=1.22.4 <2"
   },
   "dependencies": {
-    "@angular/cli": "8.3.17",
+    "@angular/cli": "8.3.28",
     "flat": "5.0.0",
-    "ramda": "0.26.1",
+    "ramda": "0.27.0",
```
Comment (peter-vasarhelyi) [inline package.json:34]: btw can we remove this? this is the only place where we use it and if I'm right we could refactor it easily to `text.split(/%d|%s/gi)`: https://github.com/emartech/angular-translate/blob/0054a67ace9eea0653f8e4c532c5fde8fa51a594/lib/service.ts#L60

Comment (peter-vasarhelyi) [reply package.json:34]: like here: https://github.com/emartech/angular-translate/blob/0054a67ace9eea0653f8e4c532c5fde8fa51a594/lib/service.ts#L30

Comment (rudywaltz) [reply package.json:34]: we can try

**Pull request merged**